lib/resources/StagedSalesforceData.ts

```diff
@@ -0,0 +1,49 @@
+import * as Promise from 'bluebird';
+
+import {
+    HudAiListAttributes,
+    HudAiUpdateAttributes,
+    Resource
+} from '../utils/Resource';
+import { RequestManager } from '../RequestManager';
+import { StagedSalesforceData } from '../entities';
+
+export interface StagedSalesforceDataListAttributes extends HudAiListAttributes {
+    userId?: string;
```

Comment:
can't remember what we agreed on for the back-end - is `userId` required?

Reply:
As of now, no. It first checks if you have `req.auth.userId` then `req.(params | body).userId`.

Reply:
So you don't have to necessarily provide it in the params/body.